lib/constable_web/views/shared_view.ex

```diff
@@ -16,6 +16,10 @@ defmodule ConstableWeb.SharedView do
     gravatar_url(user.email, secure: true)
   end
 
+  def datetime_string(datetime) do
+    datetime |> NaiveDateTime.to_iso8601
```

Comment:
I think it would be preferable to write this as `NaiveDateTime.to_iso8601(datetime)` to avoid the pipe operator when there's only one value.

Reply:
will do